src/Instances/New.lua

```diff
@@ -61,7 +62,7 @@ local function New(className: string)
 		]]
 		for key, value in pairs(propertyTable) do
 			-- ignore some keys which will be processed later
-			if key == Children or key == "Parent" then
+			if key == Children or key == Cleanup or key == "Parent" then
 				continue
 
```

Comment:
Is there a missing `end` here?

Reply:
It doesn't look like it, I will re-run tests to verify, but it should just go on to the next ``elseif`` statement rather than having an ``end``.

Tests re-run: https://github.com/nottirb/Fusion/runs/4587434381?check_suite_focus=true

Reply:
The step 2.1 comment obscures the elseif, which replaces the missing end. This might be something to consider in the Fusion formatting guidelines, because like here, it's easy to mistake while reading. Should the if block comment be after the if condition instead?